### PR TITLE
Jenkins build fix

### PR DESCRIPTION
**Pull request by sainsay**
[change] argument for build server install.bat
[fix] Jenkins now uses install.bat and build 32 and 64 bit

### CODE REVIEW

**Comment by sainsay** [comment]
Build still not fully working will notify when issues are fixed.  pull request allows me to build on Jenkins please do not close

**Comment by sainsay** [comment]
Current setup seems to be working. waiting for CMake to be updated to 3.13 on the server before merging.

**Comment by sainsay** [comment]
all issues should be fixed

File: CMakeLists.txt [hunk 1]
```diff
@@ -5,6 +5,8 @@ add_compile_definitions(_ENABLE_EXTENDED_ALIGNED_STORAGE)
 
 project(Wisp)
 
+option(ENABLE_UNIT_TEST "enables unit tests." OFF)
```
Comment (VZout) [inline CMakeLists.txt]: We likely want to rename this to WISP_UNIT_TESTS = ON/OFF
So it doesn't collide.

**Pull request merged**